types.ts

```diff
@@ -132,6 +132,11 @@ export interface User extends Account {
   upframeCalendarId?: string
 }
 
+export interface SearchQuery {
+  searchQuert?: string
+  amount?: number
+}
```

Comment:
`searchQuert` looks like a typo and it doesn't seem like the interface is used anywhere